api/sql.go

```diff
@@ -16,9 +16,9 @@ type ConnectionInfo struct {
 	Password         string  `jsonapi:"attr,password"`
 	UseEmptyPassword bool    `jsonapi:"attr,useEmptyPassword"`
 	InstanceID       *int    `jsonapi:"attr,instanceId"`
-	SslCa            string  `jsonapi:"attr,sslCa"`
-	SslCert          string  `jsonapi:"attr,sslCert"`
-	SslKey           string  `jsonapi:"attr,sslKey"`
+	SslCa            *string `jsonapi:"attr,sslCa"`
```

Comment:
Or shall we use empty string to determine existence?

Reply:
I don't think so. We need to handle it like `password` field. And we have `UseEmptyPassword` field but not something like it of SSL.

Reply:
It's enough for release. We can explore a better way to make the logic easier next week.